src/views/Actors/components/ActorTable/ActorTable.tsx

```diff
@@ -36,9 +36,9 @@ const TableWrapper = styled.div({
 const ContainerList = styled.div({
   display: 'flex',
   flexDirection: 'column',
-  gap: 24,
+  gap: 8,
   [lightTheme.breakpoints.up('tablet_768')]: {
```

Comment:
Once changed to `@material` styles, we should use `theme` instead of `lightTheme` here

Reply:
done